content/news/2017-01-stats-post.md

```diff
@@ -0,0 +1,71 @@
+![](/images/news/20170110-stats-enable-sysinfo-checkbox.png)
+
+If you like this post and happen to have OpenRA configured so that is doesn't send system information,<br /> you can easily re-enable it in the "Advanced" tab of the settings dialog.
+</div>
+
+When the game is allowed to send system information, the time of the last update is recorded in our database. Thanks to this, we know that about 5,000 users have played OpenRA in the first week of 2017. This also allows us to see how "loyal" players are to OpenRA in general:
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-last-launches.png)
+
+It is an unfathomable idea, but it sure looks like there are users who don't play OpenRA every day!
+</div>
+
+We can also get a general idea of the composition of our player base by looking at the language settings that get sent back to us. As expected, the vast majority of players use English (~57%) as their primary language, followed by German-speaking people (16%), which combined make up almost three-fourth of all our users. Surprisingly, the relatively small Netherlands come in at third place with 4.5% of the total. Also noteworthy is a quiet Chinese minority of ~2% at seventh place.
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-languages.png)
+</div>
+
+It is no big secret that Red Alert is the most popular mod, but just by how much? Here's the answer:
+
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-mod-popularity.png)
+
+Since system information is also sent by people who are not playing online matches, the obvious conclusion here is that Tiberian Dawn and Dune 2000 players are much more invested in single-player and/or skirmish games against the AI.
+</div>
+
+Another interesting way to look at this data is to find out which mods are popular in which countries:
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-mod-popularity-per-language.png)
+
+Dune 2000 has a relatively larger following in Russia, the Czech Republic, Poland and Italy,<br /> while the Scandinavian countries are deep in the Red Alert camp.<br /> Tiberian Dawn is stronger in Hungary and Italy.
+</div>
+
+The OpenRA master server also records the maps on which online matches are played. The top ten most popular ones are these:
+<div style="text-align:center" markdown="1">
+![](/images/news/20170110-stats-top-ten-mp-maps.png)
+</div>
+
+Here is the complete list of the top 40 (well, top 38) maps, all those with more than 250 plays. Red Alert's dominance is again felt here, as every single one of those maps belongs to that mod.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20170110-stats-most-popular-maps.png" height="300" width="600" />
+
+Personally, I find immense joy in the fact that two of the three multiplayer cooperative missions<br /> that have been added in the last release found a spot in the top 40!
```

Comment:
Hooray! https://github.com/OpenRA/OpenRA/issues/5490 was worth it then. Also ping to @ScottNZ!